vector/src/main/res/values/strings.xml

```diff
@@ -425,6 +425,8 @@
     <string name="permissions_msg_contacts_warning_other_androids">Element can check your address book to find other Matrix users based on their email and phone numbers.\n\nDo you agree to share your address book for this purpose?</string>
 
     <string name="permissions_action_not_performed_missing_permissions">Sorry. Action not performed, due to missing permissions</string>
+    <string name="permissions_denied_qr_code">To scan a QR code, allow Camera permission to take a picture</string>
```

Comment:
I would remove "to take a picture" because the text xill be truncated in the Snackbar, and you have two rationale: "To scan a QR code"

So maybe better to write:
"Please grant the Camera permission to scan a QR code"

Reply:
How about "To scan a QR code, you need to allow camera access"?  starting with the QR code bit seems a bit easier to understand, for me